README.md

```diff
@@ -36,19 +36,16 @@ submitting a pull request you are agreeing to comply with this waiver of
 copyright interest.
 
 ## License Standard Notice
-The repository utilizes code licensed under the terms of the Apache Software
-License and therefore is licensed under ASL v2 or later.
-
-This source code in this repository is free: you can redistribute it and/or modify it under
-the terms of the Apache Software License version 2, or (at your option) any
-later version.
+This project is in the public domain within the United States, and copyright and
+related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+All contributions to this project will be released under the CC0 dedication. By
+submitting a pull request or issue, you are agreeing to comply with this waiver
+of copyright interest and acknowledge that you have no expectation of payment,
+unless pursuant to an existing contract or agreement.
 
-This source code in this repository is distributed in the hope that it will be useful, but WITHOUT ANY
-WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-PARTICULAR PURPOSE. See the Apache Software License for more details.
-
-You should have received a copy of the Apache Software License along with this
-program. If not, see http://www.apache.org/licenses/LICENSE-2.0.html
+The repository utilizes code licensed under the terms of the Apache Software
+License, which if unmodified, remains licensed under ASL v2 or later, but all other code
+contributed to this project falls under the CC0 1.0 Universal public domain dedication.
```

Comment:
Seems like something that should go with this is a) ensuring that we reliably call out non-public-domain files in the repo with comments (e.g. [`prime-router/gradlew.bat`](https://github.com/CDCgov/prime-reportstream/blob/518bf88991f42ebf0d3aee40b00f2567e6d59797/prime-router/gradlew.bat#L1-L15)) and b) explain here that such files can be identified with that header. Otherwise I’m not sure what a consumer should do other than treat the whole codebase as *if* it were ASL2, since it’s the more restrictive license.

Reply:
I thought the CC0 is more restrictive because it prohibits people from taking the work in the repo and creating their own patentable work off of it, but I could be mistaken. I will check with the lawyers and see what they think.

Reply:
Ah, yeah, I guess this is more just *differently* restrictive; I was focusing on copyright. So someone would still have to treat everything as the most-restrictive-common-denominator.

> CC0 is more restrictive because it prohibits people from taking the work in the repo and creating their own patentable work off of it

Worth checking with the lawyers, but I don’t believe it does that. A contributor can patent their contributions under CC0, potentially preventing a consumer from using it (so I guess it is potentially more restrictive, in that sense) but it doesn’t address what consumers can do with the work, like incorporating it into a patented invention (but at that point, I don’t know what patent law says about inventions that involve public domain stuff). ASL2 explicitly says consumers are granted a license to use it even if some contributions are patented, but also makes no claims about downstream patenting by consumers (but I imagine that wouldn’t be possible since copyright is retained in ASL2, which I think would complicate patent applications by non-copyright-holders).

Reply:
Anyway, I think my main concern was that this paragraph throws things back into an unclear status in terms of “what can I do with the contents of this repository?” and that still stands.